Sources/XcodeProj/Extensions/Dictionary+Extras.swift

```diff
@@ -4,6 +4,8 @@ import Foundation
 ///
 /// - Parameter path: the path of the .plist file.
 /// - Returns: initialized dictionary.
-public func loadPlist(path: String) -> [String: AnyObject]? {
-    NSDictionary(contentsOfFile: path) as? [String: AnyObject]
+public func loadPlist(path: String) -> [String: PlistObject]? {
+    let fileURL = URL(fileURLWithPath: path)
+    guard let data = try? Data(contentsOf: fileURL) else { return nil }
+    return try? PropertyListDecoder().decode([String: PlistObject].self, from: data)
```

Comment:
Currently this is just used in tests and most of the logic is around `nil` checking.  I'm pushing a change so this isnt public since `PlistObject` doesnt need to be public either.